src/shared/copy.c

```diff
@@ -824,6 +829,9 @@ static int fd_copy_regular(
         if (fdt < 0)
                 return -errno;
 
+        if (attrs != 0)
+                (void) chattr_full(fdt, NULL, attrs, CHATTR_EARLY_FL, NULL, NULL, CHATTR_FALLBACK_BITWISE);
```

Comment:
this sounds risky. I think this should be opt-in via a flag. some bits have such weird semantics, and translate really badly between file systems.